global-attributes.yml

```diff
@@ -0,0 +1,75 @@
+    # asciidoctor prefix for section IDs
+    idprefix: ''
+    idseparator: '-'
+    # Antora User Interface Macros (btn, menu)
+    experimental: true
+    # Antora pagination (prev page, next page)
+    page-pagination: true
+    # Custom attribute page-component-build-list
+    # Only embed named opengraph images in docs-ui via src/partials/head-meta-opengraph.hbs sourced from docs/overlay
+    page-component-build-list: 'docs, docs-main, ocis, server, webui, desktop, android, ios-app, branding'
+#   common
+    docs-base-url: 'https://doc.owncloud.com'
+    oc-complete-base-url: 'https://download.owncloud.com/server/stable'
+    oc-contact-url: 'https://owncloud.com/contact/'
+    oc-help-url: 'https://owncloud.com/docs-guides/'
+    oc-marketplace-url: 'https://marketplace.owncloud.com'
+    oc-central-url: 'https://central.owncloud.org'
+    oc-support-url: 'https://owncloud.com/support'
+#
+# note that all attributes ending with '@' are used for versioned docs. the content can be
+# overwritten by the build process from the corresponding antory.yml file if populated - do not change
+#
+#   docs
+    latest-docs-version: 'next'
+    previous-docs-version: 'next'
+#   server
+    latest-server-version: '10.15'
+    latest-server-download-version: '10.15.0'
+    previous-server-version: '10.14'
+    current-server-version: '10.15'
+    oc-changelog-url: 'https://owncloud.com/changelog/server/'
+    oc-install-package-url: 'https://download.owncloud.com/server/stable/?sort=time&order=asc'
+    oc-examples-server-url: 'https://owncloud.install.com/owncloud'
+    oc-examples-server-ip: '127.0.0.1'
+    oc-examples-username: 'username'
+    oc-examples-password: 'password'
+    oc-complete-name: 'owncloud-complete-latest'
+    occ-command-example-prefix: 'sudo -u www-data ./occ'
+    occ-command-example-prefix-no-sudo: 'occ'
+    php-net-url: 'https://www.php.net'
+    php-supported-versions-url: 'https://www.php.net/supported-versions.php'
+    http-status-codes-base-url: 'https://developer.mozilla.org/en-US/docs/Web/HTTP/Status'
+    std-port-http: '8080'
+    std-port-memcache: '11211'
+    std-port-mysql: '3306'
+    std-port-redis: '6379'
+#   ocis
+    # branch versions
+    latest-ocis-version: '7.0'
+    previous-ocis-version: '5.0'
+    # Versions mainly for printing like in docs-main release info and in docs-ocis to define the latest production version.
+    # Versions in the ocis docs need to be defined in the branch specific docs-ocis/antora.yaml file.
+    # To do so, change the values in the branch of docs-ocis/antora.yml like service_xxx and compose_xxx.
+    ocis-actual-version: '7.0.0'
+    ocis-former-version: '5.0.9'
+    # Needed in docs-ocis to define which rolling release to print like in the envvars table or in deployment examples
+    ocis-rolling-version: '6.6.1'
+    ocis-compiled: '2024-12-17 00:00:00 +0000 UTC'
+    ocis-downloadpage-url: 'https://download.owncloud.com/ocis/ocis'
```

Comment:
Do we need all this in here?
docs-server is just for the oC10 core, apps, classic UI etc.

Reply:
yes, because if we phase out server from the docs in the future and archive the repo, everything is complete and we do not forget this file to add if one wants to build it locally.